doc/user_guide/lfric.rst

```diff
@@ -879,7 +879,8 @@ types.
 Rules specific to General-Purpose Kernels without CMA Operators
 +++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 
-1) General-purpose kernels with ``operates_on = CELL_COLUMN`` accept
+1) General-purpose kernels with ``operates_on = CELL_COLUMN`` or
+   ``OWNED_CELL_COLUMN`` accept
```

Comment:
`General-purpose kernels that operate in any kind of CELL_COLUMN`?(or does this not apply to owned_and_halo_cell_colum and halo_cell_colum)